packages/web/modals/validator-next-step.tsx

```diff
@@ -0,0 +1,92 @@
+import { Staking } from "@keplr-wallet/stores";
+import { observer } from "mobx-react-lite";
+import Link from "next/link";
+import { FunctionComponent } from "react";
+import { useTranslation } from "react-multi-lang";
+
+import { Button } from "~/components/buttons";
+import { ModalBase, ModalBaseProps } from "~/modals/base";
+
+interface ExtendedModalBaseProps extends ModalBaseProps {
+  usersValidatorsMap: Map<string, Staking.Delegation>;
+  setShowValidatorModal: (val: boolean) => void;
+}
+
+export const ValidatorNextStepModal: FunctionComponent<ExtendedModalBaseProps> =
+  observer((props) => <ValidatorSquadContent {...props} />);
+
+const ValidatorSquadContent: FunctionComponent<ExtendedModalBaseProps> =
+  observer(
+    ({ onRequestClose, isOpen, usersValidatorsMap, setShowValidatorModal }) => {
+      // i18n
+      const t = useTranslation();
+
+      const isNewUser = usersValidatorsMap.size === 0;
```

Comment:
@mattupham we should touch base with @syed8rahman next week on this. Presumably, we don't want to show the new modal every time someone without validators comes to the page. We should probably store some key in indexedDB. But let's confirm with @syed8rahman first